nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -168,37 +171,52 @@ a.anchor-link {
 </div>
 </div>
 
-<script src="{{resources.reveal.url_prefix}}/lib/js/head.min.js"></script>
-
-<script src="{{resources.reveal.url_prefix}}/js/reveal.js"></script>
-
 <script>
 
-// Full list of configuration options available here: https://github.com/hakimel/reveal.js#configuration
-Reveal.initialize({
-controls: true,
-progress: true,
-history: true,
-
-theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
-transition: Reveal.getQueryHash().transition || 'linear', // default/cube/page/concave/zoom/linear/none
-
-// Optional libraries used to extend on reveal.js
-dependencies: [
-{ src: "{{resources.reveal.url_prefix}}/lib/js/classList.js", condition: function() { return !document.body.classList; } },
-{ src: "{{resources.reveal.url_prefix}}/plugin/notes/notes.js", async: true, condition: function() { return !!document.body.classList; } }
-]
-});
-</script>
-
-<!-- Loading mathjax macro -->
-{{ mathjax() }}
+require(
+    {
+      // for really large notebook this probably makes sense
+      waitSeconds: 30
```

Comment:
You are right... I was working on RISE at the same time, where the time to load the live notebook could affect the reveal.js load and arise a timeout...
Certainly, I don't think this setting is needed here, unless I am missing something else... so I will delete it for now...